lib/rust/src/lib.rs

```diff
@@ -8,8 +8,8 @@ pub use ethers_core as core;
 
 pub use rpc::{call_contract, get_provider, request};
 
-#[ic_cdk_macros::query(name = "__transform_ic_evm_rpc")]
-pub fn transform_evm_rpc(args: TransformArgs) -> HttpResponse {
+#[ic_cdk_macros::query(name = "__ic_evm_transform_json_rpc")]
```

Comment:
This is an artifact from including a Rust library that I was originally using for the JSON-RPC logic. Since I ended up separating the library and canister implementations, `__ic_evm_transform_json_rpc` is provided by the `ic_evm` library whereas `__transform_json_rpc` is specific to the canister. They're essentially doing the same thing, so I used (arbitrarily) different names to distinguish these from each other.